prow/plugins/milestonestatus/milestonestatus.go

```diff
@@ -31,7 +31,7 @@ const pluginName = "milestonestatus"
 
 var (
 	statusRegex   = regexp.MustCompile(`(?m)^/status\s+(.+)$`)
-	mustBeSigLead = "You must be a member of the @kubernetes/kubernetes-milestone-maintainers github team to add status labels."
+	mustBeSigLead = "You must be a member of the kubernetes/kubernetes-milestone-maintainers github team to add status labels."
```

Comment:
Since this will remove the hyperlink you should make this a markdown link to https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers/members

This should also be configurable instead of being hardcoded to be kubernetes specific, but that doesn't need to be done here. I'll open an issue.

Reply:
I've updated to a link.

Reply:
ref #5276